src/services/expenses/expenses.js

```diff
@@ -0,0 +1,99 @@
+'use strict';
+
+const { v4: uuid } = require('uuid');
+
+const { db } = require('../../db');
+
+const { validateExpenseData } = require('./validation');
+
+const resetStore = async () => {
+  await db`DROP TABLE IF EXISTS expenses`;
+
+  await db`CREATE TABLE expenses (
+    id UUID PRIMARY KEY,
+    userId UUID,
+    title TEXT,
+    category TEXT,
+    amount NUMERIC,
+    spentAt TIMESTAMP,
+    note TEXT
+  )`;
+};
+
+const getAll = async ({ userId, categories, from, to }) => {
```

Comment:
It's a common design mistake when the Service is tightly coupled with the DB.
Better to introduce a Repository concept, ideally with dependency injection.